### ISSUE TITLE
Safari on iOS can get stuck in a microphone request loop

### ISSUE THREAD

**Issue by brianpeiris** [mozilla/hubs#816]
Reported here: https://twitter.com/Just_Janes/status/1080654055532183552

**Comment by brianpeiris** [comment]
Possibly occurs when clicking on a hubs link that opens in a Safari WebView?

**Comment by VestigeJ** [comment]
‪Just checked through my settings for iOS it’s definitely my personal settings. I actually forgot I turned camera/microphone access off on purpose for Safari (privacy paranoid). This lead to the page probably being unable to request access in the first place. 

iPhone X
iOS/Safari 12.1.2

**Comment by misslivirose** [comment]
I'm changing this to an enhancement because it sounds like this is expected if the browser is unable to request permissions. That said, I think the improvement here is to better message to the user that they had denied permissions and to check the settings, so changes to that message would probably be an easy win.

**Comment by larsbergstrom** [comment]
Yes, an internal user and I both just ran into this and right now it's pretty hard to know that you have denied microphone/camera permissions to all web sites in Safari. It would be great to have a message, as right now everything is just disabled and there's no messaging about why / what to do in the flow.

**Comment by takahirox** [comment]
Yeah, I think the solution should be clarifying the message. Updating the message is easy so I added "good first issue" label.

And related, we should verify if the mic permission must still be allowed on Safari to enter a room https://github.com/mozilla/hubs/issues/4019#issuecomment-931587950
